source/retryable-writes/tests/README.md

```diff
@@ -106,17 +106,197 @@ Drivers should test that transactions IDs are always included in commands for su
 
 The following tests ensure that retryable writes work properly with replica sets and sharded clusters.
 
-1. Test that retryable writes raise an exception when using the MMAPv1 storage engine. For this test, execute a write
-   operation, such as `insertOne`, which should generate an exception. Assert that the error message is the replacement
-   error message:
+- Test that retryable writes raise an exception when using the MMAPv1 storage engine. For this test, execute a write
```

Comment:
This should be an ordered list so the prose tests remain numbered.

Alternatively, it may be more readable to use header tags with manual numbering:

```
### 1. Retryable writes error when using MMAPv1 storage engine
```

That approach is more common in test READMEs with more than just a couple of prose tests, and this file now has five. If you go this route I'd just suggest shortening the name a bit (or leaving the original text as-is and adding your own header above).

Bonus: if you go this route, the `[!NOTE]` syntax below will start working because it will no longer be nested within a list item.

Reply:
Done